tests/pytest.ini

```diff
@@ -3,5 +3,5 @@
 env =
     CENTML_CACHE_DIR=/tmp/centml-test
 markers = 
-    gpu: need the gpu to run quickly
+    gpu: this test needs the gpu to run quickly
```

Comment:
can we also turn off the warnings? I tried running the tests and was getting a lot of warnings.

Reply:
Do you want to turn off warnings for all tests or just the gpu tests?

also could you please paste your warnings? I don't have any on my end so I wonder what they are

Reply:
```
(env) [anandj@fedora tests]$ pytest --sanity
=========================================================================================================================================================================================================== test session starts ============================================================================================================================================================================================================
platform linux -- Python 3.9.13, pytest-7.4.3, pluggy-1.3.0
rootdir: /home/anandj/data/code/centml-python-client/tests
configfile: pytest.ini
plugins: anyio-3.7.1
collected 26 items                                                                                                                                                                                                                                                                                                                                                                                                                         

test_backend.py ..............                                                                                                                                                                                                                                                                                                                                                                                                       [ 53%]
test_server.py ......ss....                                                                                                                                                                                                                                                                                                                                                                                                          [100%]

============================================================================================================================================================================================================= warnings summary =============================================================================================================================================================================================================
../env/lib/python3.9/site-packages/_pytest/config/__init__.py:1373
  /home/anandj/data/code/centml-python-client/env/lib/python3.9/site-packages/_pytest/config/__init__.py:1373: PytestConfigWarning: Unknown config option: env
  
    self._warn_or_fail_if_strict(f"Unknown config option: {key}\n")

../env/lib/python3.9/site-packages/huggingface_hub/inference/_text_generation.py:121
  /home/anandj/data/code/centml-python-client/env/lib/python3.9/site-packages/huggingface_hub/inference/_text_generation.py:121: PydanticDeprecatedSince20: Pydantic V1 style `@validator` validators are deprecated. You should migrate to Pydantic V2 style `@field_validator` validators, see the migration guide for more details. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.5/migration/
    @validator("best_of")

../env/lib/python3.9/site-packages/huggingface_hub/inference/_text_generation.py:140
  /home/anandj/data/code/centml-python-client/env/lib/python3.9/site-packages/huggingface_hub/inference/_text_generation.py:140: PydanticDeprecatedSince20: Pydantic V1 style `@validator` validators are deprecated. You should migrate to Pydantic V2 style `@field_validator` validators, see the migration guide for more details. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.5/migration/
    @validator("repetition_penalty")

../env/lib/python3.9/site-packages/huggingface_hub/inference/_text_generation.py:146
  /home/anandj/data/code/centml-python-client/env/lib/python3.9/site-packages/huggingface_hub/inference/_text_generation.py:146: PydanticDeprecatedSince20: Pydantic V1 style `@validator` validators are deprecated. You should migrate to Pydantic V2 style `@field_validator` validators, see the migration guide for more details. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.5/migration/
    @validator("seed")

../env/lib/python3.9/site-packages/huggingface_hub/inference/_text_generation.py:152
  /home/anandj/data/code/centml-python-client/env/lib/python3.9/site-packages/huggingface_hub/inference/_text_generation.py:152: PydanticDeprecatedSince20: Pydantic V1 style `@validator` validators are deprecated. You should migrate to Pydantic V2 style `@field_validator` validators, see the migration guide for more details. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.5/migration/
    @validator("temperature")

../env/lib/python3.9/site-packages/huggingface_hub/inference/_text_generation.py:158
  /home/anandj/data/code/centml-python-client/env/lib/python3.9/site-packages/huggingface_hub/inference/_text_generation.py:158: PydanticDeprecatedSince20: Pydantic V1 style `@validator` validators are deprecated. You should migrate to Pydantic V2 style `@field_validator` validators, see the migration guide for more details. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.5/migration/
    @validator("top_k")

../env/lib/python3.9/site-packages/huggingface_hub/inference/_text_generation.py:164
  /home/anandj/data/code/centml-python-client/env/lib/python3.9/site-packages/huggingface_hub/inference/_text_generation.py:164: PydanticDeprecatedSince20: Pydantic V1 style `@validator` validators are deprecated. You should migrate to Pydantic V2 style `@field_validator` validators, see the migration guide for more details. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.5/migration/
    @validator("top_p")

../env/lib/python3.9/site-packages/huggingface_hub/inference/_text_generation.py:170
  /home/anandj/data/code/centml-python-client/env/lib/python3.9/site-packages/huggingface_hub/inference/_text_generation.py:170: PydanticDeprecatedSince20: Pydantic V1 style `@validator` validators are deprecated. You should migrate to Pydantic V2 style `@field_validator` validators, see the migration guide for more details. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.5/migration/
    @validator("truncate")

../env/lib/python3.9/site-packages/huggingface_hub/inference/_text_generation.py:176
  /home/anandj/data/code/centml-python-client/env/lib/python3.9/site-packages/huggingface_hub/inference/_text_generation.py:176: PydanticDeprecatedSince20: Pydantic V1 style `@validator` validators are deprecated. You should migrate to Pydantic V2 style `@field_validator` validators, see the migration guide for more details. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.5/migration/
    @validator("typical_p")

../env/lib/python3.9/site-packages/huggingface_hub/inference/_text_generation.py:204
  /home/anandj/data/code/centml-python-client/env/lib/python3.9/site-packages/huggingface_hub/inference/_text_generation.py:204: PydanticDeprecatedSince20: Pydantic V1 style `@validator` validators are deprecated. You should migrate to Pydantic V2 style `@field_validator` validators, see the migration guide for more details. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.5/migration/
    @validator("inputs")

../env/lib/python3.9/site-packages/huggingface_hub/inference/_text_generation.py:210
  /home/anandj/data/code/centml-python-client/env/lib/python3.9/site-packages/huggingface_hub/inference/_text_generation.py:210: PydanticDeprecatedSince20: Pydantic V1 style `@validator` validators are deprecated. You should migrate to Pydantic V2 style `@field_validator` validators, see the migration guide for more details. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.5/migration/
    @validator("stream")

test_backend.py::TestGetModelId::test_exception_on_save_graph_failure
  /home/anandj/data/code/centml-python-client/env/lib64/python3.9/site-packages/torch/overrides.py:110: UserWarning: 'has_cuda' is deprecated, please use 'torch.backends.cuda.is_built()'
    torch.has_cuda,

test_backend.py::TestGetModelId::test_exception_on_save_graph_failure
  /home/anandj/data/code/centml-python-client/env/lib64/python3.9/site-packages/torch/overrides.py:111: UserWarning: 'has_cudnn' is deprecated, please use 'torch.backends.cudnn.is_available()'
    torch.has_cudnn,

test_backend.py::TestGetModelId::test_exception_on_save_graph_failure
  /home/anandj/data/code/centml-python-client/env/lib64/python3.9/site-packages/torch/overrides.py:117: UserWarning: 'has_mps' is deprecated, please use 'torch.backends.mps.is_built()'
    torch.has_mps,

test_backend.py::TestGetModelId::test_exception_on_save_graph_failure
  /home/anandj/data/code/centml-python-client/env/lib64/python3.9/site-packages/torch/overrides.py:118: UserWarning: 'has_mkldnn' is deprecated, please use 'torch.backends.mkldnn.is_available()'
    torch.has_mkldnn,

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
================================================================================================================================================================================================ 24 passed, 2 skipped, 15 warnings in 4.66s ================================================================================================================================================================================================
```

Reply:
Oh right. I forgot I fixed these locally. I will surpress warnings